blog/_posts/2023-05-30-new-class-tscatter.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "New class TScatter"
+layout: archive
+author: Olivier Couet
+---
+
+<center>
+   <img src="{{'/assets/images/scatter.png' | relative_url}}" alt="TScatter" style="width: 60%" width="1600" height="749" />
+</center>
+
+2D Scatter plots are a very popular way to represent scientific data. [Many scientific plotting
+packages](https://matplotlib.org/stable/gallery/shapes_and_collections/scatter.html#sphx-glr-gallery-shapes-and-collections-scatter-py)
+have this functionality. ROOT itself offers this kind of tools via:
+
+  - [The option `SCAT` to draw 2D histograms](https://root.cern/doc/master/classTHistPainter.html#HP11).
+    But that's indeed a fake scatter plot because the points drawn are random in each bins.
+    The number of points being proportional to the cell content.
+  - [The option `P` to draw TGraph](https://root.cern/doc/master/classTGraphPainter.html#GrP1).
+    A marker will be drawn at each point positions but all he marker will all have the same size and the same color.
```

Comment:
Should be: A marker will be drawn at each point positions but all markers will have the same size and the same color